README.md

```diff
@@ -1,154 +1,156 @@
-needs to be bound to.
+* The app author (developer) creates a `ServiceBindingRequest` and specifies:
+  * The resource that needs the binding information. The resource can be specified by label
+    selectors;
+  * The backing service's resource reference that the imported application needs to be bound to;
 
 * The Service Binding Controller then:
-  * Reads backing service operator OLM descriptor to discover the binding
-    attributes
-  * Creates a binding secret for the backing service, example, 
-    an operator-managed database.
-  * Injects environment variables into the applications's DeploymentConfig,
-    Deployment or Replicaset
+  * Reads backing service operator OLM descriptor to discover the binding attributes
+  * Creates a binding secret for the backing service, example, an operator-managed database;
+  * Injects environment variables into the applications's `DeploymentConfig`, `Deployment` or
+    `Replicaset`;
 
-Here is an example of the "bind-able" operator OLM Descriptor - in this case 
-for a PostgreSQL database backing operator:
+Here is an example of the *bind-able* operator OLM Descriptor -- in this case for a PostgreSQL
+database backing operator:
 
-```yaml:
+``` yaml
+---
+[...]
 statusDescriptors:
   description: Name of the Secret to hold the DB user and password
     displayName: DB Password Credentials
     path: dbCredentials
-
     x-descriptors:
       - urn:alm:descriptor:io.kubernetes:Secret
       - urn:alm:descriptor:servicebindingrequest:env:object:secret:user
       - urn:alm:descriptor:servicebindingrequest:env:object:secret:password
-
   description: Database connection IP address
     displayName: DB IP address
     path: dbConnectionIP
     x-descriptors:
       - urn:alm:descriptor:servicebindingrequest:env:attribute
-      - description: Database connection port
-        displayName: DB port
-        path: dbConnectionPort
-        x-descriptors:
-      - urn:alm:descriptor:servicebindingrequest:env:attribute
 ```
 
 ## Quick Start
- 
- Clone the repository and run `make local` in an existing 
- `kube:admin` openshift CLI session.
-
- Alternatively, install the operator using
-
- ```apiVersion: operators.coreos.com/v1
-     kind: OperatorSource
-     metadata:
-       name: redhat-developer-operators
-       namespace: openshift-marketplace
-     spec:
-       type: appregistry
-       endpoint: https://quay.io/cnr
-       registryNamespace: redhat-developer
-   ```
+
+Clone the repository and run `make local` in an existing `kube:admin` openshift CLI session.
+Alternatively, install the operator using:
+
+``` bash
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: redhat-developer-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: redhat-developer
+EOS
+```
 
 ## Example Scenario
 
 *What the Cluster Admin does*
```

Comment:
Emphasis [should not be used](https://github.com/DavidAnson/markdownlint/blob/v0.15.0/doc/Rules.md#md036---emphasis-used-instead-of-a-heading) as heading we sholld use 

```md
### What the Cluster Admin does
```

instead

Reply:
I would also prefer this, way. (cc @ldimaggi)